dashboard/src/assets/styles/_bootstrap-overrides.scss

```diff
@@ -1,6 +1,15 @@
 $grid-gutter-width: 30px;
 
-$font-size-base: 16px;
+$font-size-caps: 13px;
+$font-size-code: 13px;
```

Comment:
Same re: reusing variable sizes. Let's assign variable names when two variables have the same value, e.g.:

```sass
$font-size-caps: 13px;
$font-size-code: $font-size-caps;
...
$font-size-nav: 16px;
$font-size-form-section-title: $font-size-nav;
...
$font-size-section-title: 18px;
$font-size-btn-lg: $font-size-section-title;
```